english/use-the-fork-luke/use-the-fork.md

```diff
@@ -67,7 +67,7 @@ this freaks you out, but you feel paralyzed, unable to move, a force grips you a
 the figure inches closer to you. You begin to realize that the the being is none 
 other than Man-Bear-Pig!
 
-You scream out of fright, and notice the reflection of the hippy holding a large a 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.